packages/babel-preset-default/package.json

```diff
@@ -36,11 +36,9 @@
 		"@babel/runtime": "^7.4.4",
 		"@wordpress/babel-plugin-import-jsx-pragma": "file:../babel-plugin-import-jsx-pragma",
 		"@wordpress/browserslist-config": "file:../browserslist-config",
+		"@wordpress/element": "file:../element",
 		"core-js": "^3.1.4"
 	},
-	"peerDependencies": {
-		"@babel/core": "^7.0.0"
-	},
```

Comment:
So does this change essentially mean that packages consuming `wp-script` don't need `@babel/core` as a dependency anymore?

Reply:
At some point, it was added as a regular dependencies so it is redundant here.